rust/blockstore/src/arrow/block/delta.rs

```diff
@@ -223,7 +225,13 @@ mod test {
         let path = tmp_dir.path().to_str().unwrap();
         let storage = Storage::Local(LocalStorage::new(path));
         let cache = Cache::new(&CacheConfig::Unbounded(UnboundedCacheConfig {}));
-        let block_manager = BlockManager::new(storage, TEST_MAX_BLOCK_SIZE_BYTES, cache);
+        let network_admission_control = NetworkAdmissionControl::new(storage.clone());
```

Comment:
Needing to shove network admission control into the tests here feels like an example of why making "AdmissionControlledStorage" just another storage impl could be a nice abstraction.